functions.php

```diff
@@ -113,9 +113,31 @@ function gesso_scripts() {
 	wp_register_script( 'gessoscripts', get_template_directory_uri() . '/js/dist/scripts.min.js', array( 'jquery', 'gessocommon', 'gessomodernizr' ), filemtime( get_template_directory() . '/js/dist/scripts.min.js' ) );
 	wp_enqueue_script( 'gessoscripts' );
 
-	// Add Google Fonts.
-	wp_register_style( 'gessoFonts', 'https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,300i,400,400i,600,600i,700,700i&display=swap', array(), $gesso_version );
-	wp_enqueue_style( 'gessoFonts' );
+		/**
+	* https://developer.wordpress.org/reference/hooks/style_loader_tag/
+	* Using style_loader_tag:
+	* - Enqueue the preconnect URL as a style
+	* - Change the rel attribute
+	* - Remove the type & media attributes
+	**/
+
+	// Enqueue Google Font
+	wp_enqueue_style( 'google-fonts-preconnect', 'https://fonts.gstatic.com', false );
+	wp_enqueue_style( 'google-fonts', 'https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,300i,400,400i,600,600i,700,700i&display=swap', array( 'google-fonts-preconnect' ), null );
+
+	// Filter enqueue styles
+	function lstreng_update_enqueued_styles( $html, $handle, $href, $media ){
+		$handles = array('google-fonts-preconnect');
+		if( in_array( $handle, $handles ) ){
+			// Change enqueued style from stylesheet to preconnect
+			$html = str_replace("rel='stylesheet'", "rel='preconnect'", $html);
+			// Remove unnecessary attributes
+			$html = str_replace("type='text/css'", "", $html);
+			$html = str_replace("media='all'", "", $html);
```

Comment:
@coreylafferty this is an example of where the quote style should be reversed since in browsers attribute values should be in double quotes and not single quotes.

Reply:
@liquilife FYI see my previous comment.